index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+<header>
+    <nav class="nav-bar" >
+        <div class="logo"><span>John Doe </span> </div>
+        <div class="menu"><span><img src="/img/Union.png" alt="menu"> </span> </div>
```

Comment:
- [ ] The hamburg/menu icon is broken. Give this a fix.
![Screenshot from 2021-08-13 12-01-30](https://user-images.githubusercontent.com/65501535/129332883-8fe738d2-1280-4a4e-901d-3bd1197aecb0.png)